test/Conversion/LinalgToTpp/linalg-to-tpp.mlir

```diff
@@ -502,3 +502,25 @@ func.func @entry() -> memref<28x32xf32> {
   memref.dealloc %alloc_0 : memref<55x32xf32>
   return %alloc_1 : memref<28x32xf32>
 }
+
+// -----
+
+#map = affine_map<(d0, d1) -> (d0, d1)>
+
+// CHECK: func.func @relu_mapping_strided(
+// CHECK-SAME:  %[[ARG0:.+]]: memref<4x4xf32>,
+// CHECK-SAME:  %[[ARG1:.+]]: memref<4x4xf32, strided<[4, 1], offset: ?>>)
+func.func @relu_mapping_strided(%i: memref<4x4xf32>, %o: memref<4x4xf32, strided<[4, 1], offset: ?>>) {
+  %cst = arith.constant 0.000000e+00 : f32 
+  // CHECK: tpp.relu ins(%[[ARG0]]
+  // CHECK-SAME:  out(%[[ARG1]]
+  linalg.generic {
+    indexing_maps = [#map, #map], 
+    iterator_types = ["parallel", "parallel"]} 
+    ins(%i : memref<4x4xf32>) outs(%o : memref<4x4xf32, strided<[4, 1], offset: ?>>) {
```

Comment:
Where is the strided requirement coming from?

Reply:
we need to handle also strided memref, and the naive type checks is too tight. This is an use-case from @KavithaTipturMadhu